packages/utils/src/crypto/por/index.spec.ts

```diff
@@ -9,9 +9,9 @@ import {
 } from '.'
 import { Response } from '../../types'
 import { randomBytes } from 'crypto'
-import { SECRET_LENGTH } from './constants'
 import { deriveAckKeyShare } from './keyDerivation'
 import assert from 'assert'
+import { SECRET_LENGTH } from '../packet/constants'
```

Comment:
Not sure if this should be the same secret length. I would rather have two objects `PacketConstants` and `PorConstants`